api/src/main/java/uk/gov/dhsc/htbhf/claimant/entity/Claim.java

```diff
@@ -96,6 +90,9 @@
     @Valid
     private Claimant claimant;
 
+    @Embedded
```

Comment:
Why has this been made an embedded object?

Reply:
As there will be 3 properties (eligiilityOutcome, overrideUntil and datesOfBirth) that shouls all be present or all absent, an object to encapsulate them felt right.